server.go

```diff
@@ -936,3 +756,280 @@ func (s *Server) handlerHistory(w http.ResponseWriter, r *http.Request) {
 		s.l.Error("Error rendering template: %v", err)
 	}
 }
+
+func (s *Server) handleJikan(data *dataAddMovie, w http.ResponseWriter, r *http.Request, sourcelink string) ([]string, error) {
+
+	jikanEnabled, err := s.data.GetCfgBool("JikanEnabled", DefaultJikanEnabled)
+	if err != nil {
+		s.l.Error("Error while retriving config value 'JikanEnabled':\n %v", err)
+		s.doError(
+			http.StatusInternalServerError,
+			"something went wrong :C",
+			w, r)
+		return nil, err
+	}
+
+	s.l.Debug("jikanEnabled: %v", jikanEnabled)
+
+	if !jikanEnabled {
+		s.l.Debug("Aborting Jikan autofill since it is not enabled")
```

Comment:
This debug log can be removed as an error is being returned.  The error should be logged in the calling context.

Reply:
Will be fixed in the next commit